packages/core-modules/contracts/modules/ElectionModule.sol

```diff
@@ -37,6 +37,7 @@ contract ElectionModule is
         settings.minEpochDuration = 7 days;
         settings.maxDateAdjustmentTolerance = 7 days;
         settings.nextEpochSeatCount = 3;
+        settings.minimumActiveMembers = 2;
```

Comment:
shouldn't be configurable after initial setup?

Reply:
Yes! Good point :)

Reply:
@leomassazza put the change in a different PR because code reviewers are wonderful people and need more love in this universe :)
https://github.com/Synthetixio/synthetix-v3/pull/840/files